processor/deltatocumulativeprocessor/config.go

```diff
@@ -13,12 +13,23 @@ import (
 var _ component.ConfigValidator = (*Config)(nil)
 
 type Config struct {
-	MaxStale time.Duration `json:"max_stale"`
+	MaxStale   time.Duration `json:"max_stale"`
+	MaxStreams int           `json:"max_streams"`
 }
 
 func (c *Config) Validate() error {
 	if c.MaxStale <= 0 {
 		return fmt.Errorf("max_stale must be a positive duration (got %s)", c.MaxStale)
 	}
+	if c.MaxStreams <= 0 {
+		return fmt.Errorf("max_streams must be a positive number (got %d)", c.MaxStreams)
+	}
 	return nil
 }
+
+func createDefaultConfig() component.Config {
+	return &Config{
+		MaxStale:   5 * time.Minute,
+		MaxStreams: 0, // disable. TODO: find good default
```

Comment:
We try not to leave TODOs in code unless there is an associated issue mentioned as well.